docker/seahorn-build.Dockerfile

```diff
@@ -0,0 +1,85 @@
+#
+# Dockerfile for SeaHorn
+# This produces package in /seahorn/build
+# Arguments:
+#  - UBUNTU:     trusty, xenial
+#  - BUILD_TYPE: Debug, Release
+#
+
+ARG UBUNTU
+
+# Pull base image.
+FROM buildpack-deps:$UBUNTU
+
+ARG BUILD_TYPE
+RUN echo "Build type set to: $BUILD_TYPE"
+
+# Install deps.
+RUN \
```

Comment:
minimize number of `RUN` commands that allocate disk space. All install/download commands must under the same `RUN`